Source/WebKit2/NetworkProcess/NetworkProcess.cpp

```diff
@@ -631,6 +635,9 @@ void NetworkProcess::initializeSandbox(const ChildProcessInitializationParameter
 
 void NetworkProcess::platformLowMemoryHandler(Critical)
 {
+#ifdef __GLIBC__
+    malloc_trim(0);
```

Comment:
(I'm not sure if I'm supposed to review this, but...)

IIRC, by default webkit uses bmalloc instead of system malloc for memory allocation. I think that malloc-related calls are only effective if webkit is built with -DUSE_SYSTEM_MALLOC=ON to force malloc() calls. However, I'm not sure if that's the best thing to do, as bmalloc exists for a reason (it's supposed to be more efficient than system malloc).